specification/signalr/data-plane/AzureSignalR/v1/azuresignalr.json

```diff
@@ -0,0 +1,875 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SignalR Service REST API",
+    "version": "v1"
+  },
+  "paths": {
+    "/api/v1/health": {
+      "head": {
+        "tags": [
+          "general"
+        ],
+        "summary": "Get service health status.",
+        "operationId": "HealthApiIndex_Head",
+        "responses": {
+          "200": {
+            "description": "The service is healthy"
+          }
+        }
+      }
+    },
+    "/api/v1/hubs/{hub}": {
+      "post": {
+        "tags": [
+          "signalr.v1"
+        ],
+        "summary": "Broadcast a message to all clients connected to target hub.",
+        "operationId": "HubProxyControllerV1Broadcast_Post",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "text/plain"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "hub",
+            "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "excluded",
+            "description": "Excluded connection Ids",
+            "type": "array",
```

Comment:
By default, this will be serialized as `excluded=id1,id2` (csv collectionFormat). If you have a different serialization format, please use this doc to explain the right value:
https://swagger.io/docs/specification/2-0/describing-parameters/ (section *Array and Multi-Value Parameters*)